### ISSUE TITLE
[Suggestion] General Naming Conventions

### ISSUE THREAD

**Issue by Extegral** [polyvalord/extcaves#3]
All items and blocks in your mod are named similar to this:

```
"block.extcaves.hanging_roots": "Hanging roots",
"block.extcaves.moss_green": "Dry moss",
"block.extcaves.moss_fire": "Lava moss",
"block.extcaves.moss_frozen": "Frozen moss",
"block.extcaves.effectcrystal_green": "Sealed green crystal"
```

It's not written anywhere and no one should strictly hold to it, but it's usually accepted that every word in item's/block's name is started from uppercase, unless it is article like "the" or "a". The most common example is Minecraft's default english localization, since we have "**Diamond Pickaxe**" and "**Heart of the Sea**" rather than "**Diamond pickaxe**" and "**Heart of the sea**".

I suggest you to correct your mod's localization in accordance to this, it will look better and more organized this way.

There is also a smaller change I would like to offer, that is to change "**Treasure pot - Quartz**" to something more convenient, like "**Quartz Treasure Pot**".

**Comment by polyvalord** [comment]
Thanks for reporting.
I will do necessary changes as soon as possible.

**Comment by polyvalord** [comment]
Fixed since 1.3.2 !
